site/source/guides/index.rst

```diff
@@ -12,5 +12,6 @@ Guides for integrating your client devices with your Start9 server, building you
   service-guides/index
   upgrade-hardware/index
   flashing/index
+  change-password
```

Comment:
Should this exist?  Change password seems like a guide but the actual page for this guide that I added to is under user-manual/overview.  Adding this entry results in this:
![image](https://github.com/Start9Labs/documentation/assets/106188942/e7edf1e1-b692-4f90-88c9-ef530c191c43)
Which also currently also exists under Overview:
![image](https://github.com/Start9Labs/documentation/assets/106188942/39bef66f-fd2f-4e9c-8f12-95770e96148c)

Reply:
I think this organization makes sense. We need to have a better UX in StartOS around prompting these features, especially backups.